templates/base/head_navbar.tmpl

```diff
@@ -179,7 +179,8 @@
 				</a>
 			{{end}}
 			<a class="item{{if .PageIsSignIn}} active{{end}}" rel="nofollow" href="{{AppSubUrl}}/user/login{{if not .PageIsSignIn}}?redirect_to={{.CurrentURL}}{{end}}">
-				{{svg "octicon-sign-in"}} {{ctx.Locale.Tr "sign_in"}}
+				{{svg "octicon-sign-in"}}
+				<span class="tw-ml-1">{{ctx.Locale.Tr "sign_in"}}</span>
```

Comment:
Similar layout above:

![image](https://github.com/user-attachments/assets/83945464-2c70-43e4-9a86-db323c92d48c)

Reply:
Yes, I'll admit, I disabled the self-registering functionality so do not have this button, but if it looks the same, the change should definitely be done there aswell?

Reply:
Yep, I think so.

And maybe it could be something like

```
{{svg "octicon-sign-in" 16 "tw-mr-1"}}{{ctx.Locale.Tr "sign_in"}}
```